pkgs/applications/networking/ssb/talenet/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchurl
+, fetchgit
+
+, runCommand
+, writeTextFile
+, makeDesktopItem
+, utillinux
+, python2
+
+
+, electron
+, nodejs-6_x
+# auxilary dep to scuttle-shell - needs to avail at runtime
+, systrayhelper
+}:
+
+let
+  # might become a problem with electron once this builds
+  # just a workaround anyway: https://github.com/svanderburg/node2nix/issues/106#issuecomment-418707444
+  nodejs = nodejs-6_x;
```

Comment:
Could you check if this has been [solved](https://github.com/svanderburg/node2nix/issues/106#issuecomment-442952578)?

Reply:
I tried again with @svanderburg new version but still run into ENOTCACHED (ansi-styles).

My current WIP branch is on [this](https://github.com/talenet/talenet/commit/a9d6a5f1937fe9cbf1ef5f8146a10ae410d45f93) branch of tale:net.